src/stcal/ramp_fitting/ols_fit.py

```diff
@@ -694,8 +698,10 @@ def ols_ramp_fit_single(ramp_data, buffsize, save_opt, readnoise_2d, gain_2d, we
 
     p_start = time.time()
 
+    log.info("Entering python code")
     image_info, integ_info, opt_info = ols_ramp_fit_single_python(
         ramp_data, buffsize, save_opt, readnoise_2d, gain_2d, weighting)
+    log.info("Returning from python ")
```

Comment:
The reason I included them as a logging feature because there is an option to switch between python and the C extension.  This would provide unambiguous feedback to the user which code is being selected.  But it's, also, reasonable to change these to debug logging statements.

Reply:
The selected algorithm is already logged in a couple of places, so I think debug level is probably sufficient for most users.